actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -24,7 +24,7 @@ def initialize
       # If the adapter cannot be found, this will default to the Redis adapter.
       # Also makes sure proper dependencies are required.
       def pubsub_adapter
-        adapter = (cable.fetch("adapter") { "redis" })
+        adapter = (cable.fetch("adapter", "redis"))
```

Comment:
I don't want to enable such like unsafe autocorrect.
Whether a block is redundant depends on whether the receiver is a hash.